messaging.rst

```diff
@@ -85,27 +85,80 @@ Messages
 Locked Transfer
 -----------------
 
-Cancellable and expirable :term:`transfer`. Sent by a node when a transfer is being initiated, this message adds a new lock to the list of pending locks from the sending participant node.
+A LockedTransfer is a message used to reserve tokens for a new mediated transfer.
 
-Invariants
-^^^^^^^^^^
+Preconditions
```

Comment:
Can you maybe give the conditions numbers? Like PC1 for precondition 1. Does that make sense to you?